### DISCUSSION TITLE
:toggle option string, failing.

### DISCUSSION THREAD

**Discussion by Borwe** [helix-editor/helix#10830] [General]
### Summary

the toggle command requires one to repeat the value of string twice for it to work because of a bad check I suspect.

### Reproduction Steps

 

I tried this: 

1. `hx`
2. `:toggle line-number relative`
I expected this to happen:
Line number becomes relative
Instead, this happened:
Got an error: 
<img width="940" alt="toggle error" src="https://github.com/helix-editor/helix/assets/3319843/62fdaf21-ce02-4c2a-9ac0-757f02d85bf8">

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

Windows

### Terminal Emulator

Windows Terminal

### Installation Method

source

### Helix Version

helix 24.3 (730e684d)

**Comment by pascalkuthe** [comment]
this is not the intended usecase of `:toggle` that is what `:set` is for. `:toggle` is meant for toggeling between values. For example `:toggle line-number relative absolute` will find out what the current value is and set the option to the other value
